trulens_eval/examples/experimental/streamlit_appui_example.ipynb

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        _dev=Path().cwd().parent.parent.resolve()
</pre><p>nit: dev code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Can we keep experimental examples in dev state and then make non-experimental notebooks when they mature?

Reply:
sure that makes sense